docs/_data/mesheryctlcommands/cmds.yml

```diff
@@ -911,16 +911,17 @@ model:
       description: Import models using a file or filepath directly to the meshery registry
       usage: mesheryctl model import [ file | filepath ]
       examples:
-         - import model /path/to/[file.yaml|file.json]
-         - import model /path/to/models
+         - mesheryctl import model /path/to/[file.yaml|file.json]
+         - mesheryctl import model /path/to/models
     export:
       name: export
       description: Export a model by name to the given output type (default is oci).
-      usage: mesheryctl model export [ model-name ]
+      usage: mesheryctl model export [model-name]
       examples:
```

Comment:
@leecalcote: Are these examples field really needed, it seems not to be used ?

Reply:
Hi @lekaf974, let me answer this one.

In early versions of docs, we used this as no auto-doc functionality was brought in at that time. Eventually when we brought the auto-doc function (via Cobra), the command index page displays only name and description. To answer your Q, yes, it is not being used for now as it is taken care within command doc pages.

Reply:
If you included examples in the respective golang file, then the example field in the command doc page will auto populate.